docs/_redirects

```diff
@@ -31,3 +31,24 @@
 /docs/kyma/latest/04-operation-guides/* https://kyma-project.io/#/04-operation-guides/README 301
 /docs/kyma/latest/05-technical-reference/* https://kyma-project.io/#/05-technical-reference/README 301
 /docs/kyma/latest/glossary/ https://kyma-project.io/#/glossary 301
+/docs/kyma/latest/01-overview/telemetry/README https://kyma-project.io/#/telemetry-manager/user/README 301
```

Comment:
Not sure if using absolute urls is necessary. Would be better to use relatives urls everywhere, so you can test the PR better in the PR preview.
Consider replacing `https://kyma-project.io/` with `/` in all the redirects